packages/react/src/components/sideNavigation/SideNavigation.stories.tsx

```diff
@@ -0,0 +1,184 @@
+import React from 'react';
+
+import { SideNavigation } from './SideNavigation';
+import { IconHome } from '../../icons';
+
+export default {
+  component: SideNavigation,
+  decorators: [
+    (storyFn) => (
+      <div style={{ backgroundColor: '#f5f5f5', display: 'grid', minHeight: '100vh' }}>
+        <div style={{ height: '100%', maxWidth: '400px' }}>{storyFn()}</div>
+      </div>
+    ),
+  ],
+  title: 'Components/SideNavigation',
+  parameters: {
+    controls: { expanded: true },
+  },
+  args: { defaultOpenMainLevels: [], toggleButtonLabel: 'Navigate to page', id: 'side-navigation' },
+};
+
+enum ID {
+  MAIN_LEVEL_1 = '#main-level-1',
+  MAIN_LEVEL_2 = '#main-level-2',
+  MAIN_LEVEL_3 = '#main-level-3',
+  MAIN_LEVEL_4 = '#main-level-4',
+  SUB_LEVEL_1 = '#sub-level-1',
+  SUB_LEVEL_2 = '#sub-level-2',
+  SUB_LEVEL_3 = '#sub-level-3',
+  SUB_LEVEL_4 = '#sub-level-4',
+}
+
+const labels = {
+  mainLevel1: 'Main level accordion',
+  mainLevel2: 'Main level accordion',
+  mainLevel3: 'Main level link',
+  mainLevel4: 'Main level external link',
+  subLevel1: 'Sub level link',
+  subLevel2: 'Sub level link',
+  subLevel3: 'Sub level link',
+  subLevel4: 'Sub level link',
+};
+
+const handleClick = (setActive) => (ev) => {
```

Comment:
If the handleClick is defined outside of a story, it will not be shown in storybook under the **Story** tab. Not sure if showing this would bring that much additional value though. Also it would have to repeated every time to show it in storybook, not sure if it's worth it.